src/main/resources/assets/bettergrass/lang/it_it.json

```diff
@@ -2,21 +2,21 @@
   "bettergrass.title": "BetterGrassify Config",
```

Comment:
Can you translate `Config` here?

Reply:
Idk if it sounds better translated or not, but as you wish